angular/src/app/tv-chart-container/tv-chart-container.component.ts

```diff
@@ -94,7 +94,7 @@ export class TvChartContainerComponent implements OnInit, OnDestroy {
             symbol: this._symbol,
             datafeed: new (window as any).Datafeeds.UDFCompatibleDatafeed(this._datafeedUrl),
             interval: this._interval,
-            container_id: this._containerId,
+            container: this._containerId,
```

Comment:
is it possible to replace this with container instead? I thought about removing an ability to provide a string as a container since it might be confused sometimes and asking for HTMLElement might fix this (and in all other examples as well)

Reply:
Can we change the html files we generate with the library so that the mobile examples continue to work (currently they use the [generated html](https://github.com/tradingview/charting_library/blob/master/index.html#L29))?

Reply:
> Can we change the html files we generate with the library so that the mobile examples continue to work (currently they use the generated html)?

Yeah, I think we need to update these pages internally so they will be deployed to github as well and will work out of the box?

Reply:
Yeah exactly 👍

Reply:
Cool, let's create an internal ticket to update our pages then (or we can do this when we'll remove an ability to provide a string as a container).